app/src/main/java/com/pvp/app/ui/screen/settings/Settings.kt

```diff
@@ -78,6 +85,35 @@ fun CategoryRow(
     )
 }
 
+@Composable
+fun ReminderSetting() {
```

Comment:
Whole setting should follow the design of `SettingCard`, now we have a text on the background which is hard to understand. You could adjust the accepted parameters of that component to fill your needs, but make it as abstract as possible and not orientated around your Reminder setting.